packages/context/src/lib/decorators/provide.ts

```diff
@@ -65,7 +61,10 @@ export function provide<ValueType>({
           return protoOrTarget.set.call(this, value);
         },
         init(this: ReactiveElement, value: ValueType) {
-          controllerMap.get(this)?.setValue(value);
+          controllerMap.set(
```

Comment:
This doesn't address the legacy decorator version. We need to cover that as well, somehow.

Reply:
My understanding is that the legacy one already works fine (at least it's tested in `packages/context/src/test/context-provider_test.ts`